src/System.IO.Compression/src/System.IO.Compression.builds

```diff
@@ -5,6 +5,9 @@
     <Project Include="System.IO.Compression.csproj">
       <AdditionalProperties>TargetGroup=</AdditionalProperties>
     </Project>
+    <Project Include="System.IO.Compression.csproj" Condition="'$(TargetsWindows)' == 'true'">
```

Comment:
@ericstj Should I remove this condition from here? (Based on https://github.com/dotnet/corefx/pull/5063)